runtimes/php/manifest.yaml

```diff
@@ -1,4 +1,4 @@
-# Deployment using this manifest file creates following OpenWhisk components:
+4# Deployment using this manifest file creates following OpenWhisk components:
```

Comment:
Typo? I guess the 4 in front of the hash will cause problems...

Reply:
good catch. thank you!